app/assets/javascripts/components/forms/checkBox.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import { Field } from 'redux-form';
+import { Checkbox } from 'react-bootstrap';
+
+class CheckBox extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.onCheck = this.onCheck.bind(this);
+  }
+
+  onCheck(e, checked) {
+    this.props.input.onChange(checked);
```

Comment:
Double check this but I don't think you need to create an `onCheck` listener for changing the value of of redux `Field` checkbox component.